catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/types/constants/core/DataType.java

```diff
@@ -15,22 +15,30 @@
 
 /**
  * These are the allowed values for the attribute Core#DATATYPE.
- *
- * Based on Dublin Core (http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=elements#terms-type) and extended to include other DDF supported types.
+ * <p>
+ * These generic type(s) of the resource include the Dublin Core Metadata Initiative DCMI Type
+ * Vocabulary (http://dublincore.org/documents/dcmi-type-vocabulary/) and are extended to include
```

Comment:
Ah, I misread the diff.

They are redundant with Text and Moving Image though, is there a reason to have both? It may cause confusion in the future.